doc/en/components/link-button.md

```diff
@@ -0,0 +1,208 @@
+
+```html
+<igc-link-button
+    variant="flat"
+    href="https://www.infragistics.com/webcomponentssite/components/general-getting-started.html">
+    Flat
+</igc-link-button>
+```
+
+<div class="sample-container loading" style="height: 70px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/link-button-flat">
+</iframe></div>
+
+### Outlined Button
+
+All you have to do to create an `outlined` button is to change button variant property:
+
+```html
+<igc-link-button
+    variant="outlined"
+    href="https://www.infragistics.com/webcomponentssite/components/general-getting-started.html">
+    Outlined
+</igc-link-button>
+```
+
+<div class="sample-container loading" style="height: 70px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/link-button-outlined">
+</iframe></div>
+
+### Contained Button
+
+Analogically, we can switch to `contained` variant:
+
+```html
+<igc-link-button
+    variant="contained"
+    href="https://www.infragistics.com/webcomponentssite/components/general-getting-started.html">
+    Contained
+</igc-link-button>
+```
+
+<div class="sample-container loading" style="height: 70px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/link-button-contained">
+</iframe></div>
+
+### Floating Action Button
+
+And for a floating action button, we can switch to `fab` variant: 
+
+```html
+<igc-link-button
+    variant="fab"
+    href="https://www.infragistics.com/webcomponentssite/components/general-getting-started.html">
+    Fab
+</igc-link-button>
+```
+
+<div class="sample-container loading" style="height: 70px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/link-button-fab">
+</iframe></div>
+
+### Disable Link Button
+
+The `disabled` property can be used to make a button unclickable:
+
+```html
+<igc-link-button variant="contained" disabled="true">Disabled</igc-link-button>
+```
+
+<div class="sample-container loading" style="height: 70px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/link-button-disabled">
+</iframe></div>
+
+### Download
+
+By setting a URL to `download` property, we can make the link button to download file when is clicked.
+
+```html
+<igc-link-button
+    variant="contained"
+    download="url_to_content"
+    target="_blank">
+    Download
+</igc-link-button>
+```
+
+<div class="sample-container loading" style="height: 70px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/link-button-download">
+</iframe></div>
+
+## Link Button Sizing
```

Comment:
I am not sure if we should repeat the name of the component in the sub-headers. Maybe just 'Size' is enough.